### PR TITLE
Brandon

### PR DESCRIPTION
**Pull request by branyzp**
updated routes and starting on http status codes

### CODE REVIEW

File: controllers/commentController.js [hunk 1]
```diff
@@ -6,18 +6,17 @@ const Comment = require('../models/commentSchema');
```
Comment (g0ld3nk1w1) [inline controllers/commentController.js:6]: http codes missing

Comment (branyzp) [reply controllers/commentController.js:6]: Will continue tonight!

File: models/commentSchema.js [hunk 2]
```diff
@@ -2,7 +2,8 @@ const mongoose = require('mongoose');
 const { Schema } = mongoose;
 
 const commentSchema = new Schema({
-	id: { type: Number, required: true },
```
Comment (g0ld3nk1w1) [inline models/commentSchema.js:5]: For discussion: I actually removed this id? Assumed most of the time ObjectID that is autogenerated is good enough.

If we want to have our own ID then need to have a mechanism to track and provide the next ID; used during creation

**Comment by g0ld3nk1w1** [comment]
Outstanding: http status codes for comment
The specific ID requirement, to be discussed on Monday maybe?

**Pull request merged**